services/device-registry-mongodb/src/test/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedTenantServiceTest.java

```diff
@@ -40,15 +38,16 @@
  */
 @ExtendWith(VertxExtension.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-@Timeout(value = 10, timeUnit = TimeUnit.SECONDS)
+@Timeout(value = 5, timeUnit = TimeUnit.SECONDS)
 class MongoDbBasedTenantServiceTest implements AbstractTenantServiceTest {
 
     private static final Logger LOG = LoggerFactory.getLogger(MongoDbBasedTenantServiceTest.class);
 
     private final MongoDbBasedTenantsConfigProperties config = new MongoDbBasedTenantsConfigProperties();
     private MongoDbBasedTenantService tenantService;
+    private MongoDbBasedTenantManagementService tenantManagementService;
```

Comment:
That is because the *AbstractTenantServiceTest* indeed contains test cases for *TenantService* as well as for *TenantManagementService* (which we might want to put into separate classes in the future). So I'd rather add a reference to MongoDbBasedTenantManagementService instead of removing the reference to MongoDbBasedTenantService.

Reply:
Separate classes in the future sounds good. The below _JavaDoc_ points to an invalid link as the `searchTenants` method has been moved from `MongoDbBasedTenantService` to `MongoDbBasedTenantManagementService`. I think we could update it to make it valid.

```
/**
 * Tests for {@link MongoDbBasedTenantService#searchTenants(int, int, List, List, Span)}.
 */
```

Reply:
done